common/reflect_resource.go

```diff
@@ -85,7 +85,7 @@ func StructToSchema(v interface{}, customize func(map[string]*schema.Schema) map
 }
 
 func handleOptional(typeField reflect.StructField, schema *schema.Schema) {
-	if strings.Contains(typeField.Tag.Get("json"), "omitempty") {
+	if strings.Contains(typeField.Tag.Get("json"), "omitempty") || strings.Contains(typeField.Tag.Get("tf"), "optional") {
```

Comment:
This may cause schema consistency validation errors, need to run also integration tests just to be sure

Reply:
If you afraid that something will break, then I can get back to original fix:

```
m["auto_stop_mins"].Required = false
m["auto_stop_mins"].Optional = true
```

Reply:
Required is false by default anyway. 

This is literally the same annoying case as with `num_workers`, where we have a lot of hacks around.

Reply:
No. If I remove omitempty then it’s implicitly set to Required = true during analysis of the struct… I already hit this issue :-)

Reply:
Yes. I need to check, maybe we can simplify other places with that

Reply:
there should be no struct tags for single place use